File: walletcore/src/main/java/com/mycelium/wapi/wallet/fio/FioGroup.java
package com.mycelium.wapi.wallet.fio;

import java.util.List;

import fiofoundation.io.fiosdk.models.fionetworkprovider.FIORequestContent;

public class FioGroup {

    enum Type {sent, pending}

    public Type status;
    public final List<FIORequestContent> children;

    public FioGroup(Type string, List<FIORequestContent> requestContents) {
        this.status = string;
        children = requestContents;
    }


}

File: walletcore/src/main/java/com/mycelium/wapi/wallet/fio/FioGroup.kt
package com.mycelium.wapi.wallet.fio

import fiofoundation.io.fiosdk.models.fionetworkprovider.FIORequestContent

class FioGroup(var status: Type, val children: MutableList<FIORequestContent> = mutableListOf()) {
    enum class Type(private val s: String) {
        SENT("SENT FIO REQUESTS"),
        PENDING("PENDING FIO REQUESTS");

        override fun toString(): String {
            return s
        }
    }
}